### PR TITLE
[taskproc] integration tests

### PR DESCRIPTION
**Pull request by keymone**
- makefile
- tox env
- some basic tests
- docker-compose link

### CODE REVIEW

File: task_processing/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+docs: .taskproc/bin/activate
+	.taskproc/bin/tox -e docs
+
+test: .taskproc/bin/activate
+	.taskproc/bin/tox
+
+itest:
+	.taskproc/bin/tox -e integration
+
+.taskproc/bin/activate:
```
Comment (jolynch) [inline task_processing/Makefile]: I'm a little sketched by this, but let's iterate.

File: task_processing/tox.ini [hunk 2]
```diff
@@ -17,3 +17,19 @@ deps = sphinx
...
     sphinx-build -b html -d build/doctrees source build/html
+
+
+[testenv:integration]
+changedir = tests/integration
+passenv = DOCKER_TLS_VERIFY DOCKER_HOST DOCKER_CERT_PATH
+deps =
+    docker-compose==1.7.1
+    pytest
+    pytest-bdd
```
Comment (jolynch) [inline task_processing/tox.ini]: nice

File: task_processing/tests/integration/mesos/sync.feature [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Feature: Mesos Sync runner
+
+Scenario: Running single task
+  Given mesos platform
```
Comment (Rob-Johnson) [inline task_processing/tests/integration/mesos/sync.feature]: in paasta we wrap these up into a single 'given a working platform'

File: task_processing/tests/integration/mesos/steps/mesos_steps.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    return Popen(
+        ['docker-compose', 'run', '--rm', 'zookeeper'],
+        stdout=PIPE,
+        stderr=PIPE
+    )
+
+
+@given('mesos zookeeper')
+def mesos_master():
+    return Popen(
```
Comment (Rob-Johnson) [inline task_processing/tests/integration/mesos/steps/mesos_steps.py]: you could steal the ``paasta_tools.utils._run`` function for this

Comment (keymone) [inline task_processing/tests/integration/mesos/steps/mesos_steps.py]: i thought about this but for now `Popen` is simple enough. eventually we'd just use `plumbum` as the bug referd to in `_run` docstring seems to have been fixed?

**Comment by keymone** [comment]
https://github.com/Yelp/task_processing/pull/2

**Pull request closed without merging**